pandas/core/internals/blocks.py

```diff
@@ -1349,7 +1349,9 @@ def get_values(self, dtype: DtypeObj | None = None) -> np.ndarray:
         """
         values = self.values
         if dtype == _dtype_obj:
-            values = values.astype(object)
+            # error: Incompatible types in assignment (expression has type
+            # "Union[ExtensionArray, ndarray]", variable has type "ExtensionArray")
+            values = values.astype(object)  # type: ignore[assignment]
```

Comment:
this is just a allow-redefinition error.

rather than ignoring, do

```
values: ArrayLike = self.values
```

on L1350

Reply:
done in next commit